cmd/config/internal/commands/cmdwrap_test.go

```diff
@@ -169,8 +169,8 @@ items:
       name: test
       app: nginx
     ports:
+    # This i the port.
```

Comment:
Comments are reordered here. Why?

Reply:
I think this is a bug fix. If you look at the input fixture above, the comments are in the new position, not the old:

```
- apiVersion: v1
  kind: Service
  metadata:
    name: test
    labels:
      app: nginx
      name: test
  spec:
    ports:
    # This i the port.
    - port: 8080
      targetPort: 8080
      name: http
    selector:
      app: nginx
      name: test
```